lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -68,15 +68,14 @@ def indent(count, skip1: false)
 
           def dns_lookup(name_or_ip)
```

Comment:
TIL we have a DNS lookup macro. Looks like nothing uses this so I wonder if we should deprecate it. Doesn't have to impact this PR.

Reply:
Let's keep it, this was requested macro by a RH customer: https://projects.theforeman.org/issues/16057